*** C:/home/shozo/Elixir/tfl_yolo3/extra/tensorflow_src/tensorflow/lite/tools/make/Makefile.org	Sun Nov 15 17:26:54 2020
--- C:/home/shozo/Elixir/tfl_yolo3/extra/tensorflow_src/tensorflow/lite/tools/make/Makefile.new	Sun Nov 15 16:55:08 2020
***************
*** 7,12 ****
--- 7,14 ----
  	MAKEFILE_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
  endif
  
+ MAKEFILE_DIR := ./$(shell realpath --relative-to=$(CURDIR) $(MAKEFILE_DIR))
+ 
  # Try to figure out the host system
  HOST_OS :=
  ifeq ($(OS),Windows_NT)
***************
*** 57,63 ****
  -lpthread \
  -lm \
  -lz \
! -ldl
  
  # There are no rules for compiling objects for the host system (since we don't
  # generate things like the protobuf compiler that require that), so all of
--- 59,66 ----
  -lpthread \
  -lm \
  -lz \
! -ldl \
! -lmman
  
  # There are no rules for compiling objects for the host system (since we don't
  # generate things like the protobuf compiler that require that), so all of
